adops/setting-up-prebid-for-amp-in-dfp.md

```diff
@@ -49,7 +49,25 @@ Enter the below code snippet in the **Code snippet** text area.
 {: .alert.alert-success :}
 You can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.html).
 
-{% include dev-docs/amp-creative.md %}
+{% highlight html %}
+
+    <script src="https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
```

Comment:
No need to indent the in-line code (unless we want the code in the code block to be indented)

Reply:
I think that code blocks should be aligned with the left edge of the code box.